src/libraries/System.Net.WebProxy/src/System/Net/WebProxy.cs

```diff
@@ -95,8 +95,7 @@ public bool UseDefaultCredentials
         }
 
         private static Uri? CreateProxyUri(string? address) =>
-            address == null ? null :
-            address.IndexOf("://", StringComparison.Ordinal) == -1 ? new Uri("http://" + address) :
+            address == null ? null : !address.Contains("://", StringComparison.Ordinal) ? new Uri("http://" + address) :
```

Comment:
The default for `string.Contains(string)` is ordinal, right?  So the second arg here shouldn't be necessary.

Reply:
Yup, it isn't necessary. This is a bug in the analyzer. I realized that the `Contains(string, StringComparison)` method calls through to the `IndexOf(string, StringComparison)` only after I put this PR up. I need to reduce the number of cases the analyzer tags